### ISSUE TITLE
Nativescript-ui-sidedrawer only display router-outlet's content in tkMainContent bug in IOS

### ISSUE THREAD

**Issue by nikeshjain** [telerik/nativescript-ui-feedback#852]
Hello,

We have recently upgraded our app to NativeScript v4.2 with Angular v6. Earlier we were using NativeScript v3.3 with Angular v4. The app has a common header and footer section for navigating through the app. The central area of the app loads different components as per the current menu selection. 

We have used "RadSideDrawer" control of the "nativescript-ui-sidedrawer" module in the app.component.html.

The "DrawerContent" contains the UI elements for user preferences that we need to show in the drawer when the user swipes down from the header
The "Maincontent" contains the UI elements for header, footer and the central area. The central area is kept in the page-router-outlet.

The navigation from the header-footer and the slider swiping features are working fine in android but in iOS the header-footer not showing

The above issue is faced only in iOS. In Android, everything is working fine. 

Following is the demo code for your reference.
https://play.nativescript.org/?template=play-ng&id=sgZD3d&v=5

An urgent help in solving this issue will be greatly appreciated!

Thanks in anticipation!
Regards,
Nikesh Jain

**Comment by NickIliev** [comment]
@nikeshjain indeed at this very moment the described scenario is not working. The good news is that the issue is already addressed and the fix is expected to be part of the upcoming release. Stay tuned for additional information related to the fix

**Comment by VladimirAmiorkov** [comment]
PR created and ready for test, after that you can look for a new version.

**Comment by zbranzov** [comment]
@nikeshjain We have released the fix in nativescript-ui-sidedrawer@4.3.1 . Notice that in order to work in the example you have provided, you will have to wrap the page-router in StackLayout.
```
<StackLayout>
	<page-router-outlet></page-router-outlet>
</StackLayout>
```
